Tests/AzineTestCase.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
delete this class and replace all references to `AzineTestCase` with `\PHPUnit\Framework\TestCase`